### ISSUE TITLE
Add Demo link

### ISSUE THREAD

**Issue by john--kane** [john--kane/svelteml#2]
Add john—kane.github.io/SvelteML to the readMe

**Comment by melMass** [comment]
Yes please, it's not working for me under codesandbox, it returns:
> Failed to execute 'texImage2D' on 'WebGL2RenderingContext': Tainted canvases may not be loaded.

https://codesandbox.io/s/svelteml-demo-o1ift

**Comment by melMass** [comment]
Seems related:

https://github.com/ml5js/ml5-library/issues/217#issuecomment-423860675

EDIT: Yes this works!!

**Comment by john--kane** [comment]
I've opened a specific issue to track this, https://github.com/john--kane/svelteml/issues/3

Open for input, at first glance it appears to be that the model is loaded but the module level does not trigger a reactive binding event so the condition to estimate the faces on the image is never hit. The image binding happens ok since that is not at the module level. Will look into it but will track in https://github.com/john--kane/svelteml/issues/3
